backend/src/pipeline.py

```diff
@@ -39,6 +39,10 @@ def processing_pipeline(packet):
     color_calibrate(packet, camsigs)
     spectrally_transform(packet, camsigs)
 
+    # Values for btrgb
+    xyz, lab, ciede = verify_color_transformation(packet, camsigs)
+    r_cam, RMSE = verify_spectral_transformation(packet, camsigs)
+
```

Comment:
Probably better to pack these in the btrgb file in the functions, but it’s not worth changing until we have that functionality in